docs/reference/release-notes/8.10.3.asciidoc

```diff
@@ -1,8 +1,6 @@
 [[release-notes-8.10.3]]
 == {es} version 8.10.3
 
```

Comment:
NB this release has the same known issue as previous 8.10.x releases, we'll need to add this (either now or in a future PR):

```suggestion

[[known-issues-8.10.3]]
[float]
=== Known issues

include::8.10.0.asciidoc[tag=repositorydata-format-change]
```

Reply:
thank you